_pages/home/updating-to-version-2211.35/migration.md

```diff
@@ -1,116 +1,195 @@
+Run the following command to update the local version of Angular to version 19:
 
 ```bash
 ng update @angular/cli@19 @angular/core@19 ngx-infinite-scroll@19 @ng-select/ng-select@14 @ngrx/store@19 angular-oauth2-oidc@19 --force
 git add .
 git commit -m "update angular 19 and 3rd party deps angular 19 compatible"
 ```
 
+This command also updates other third-party dependencies from the Angular ecosystem to versions that are compatible with Angular 19, such as `@ng-select/ng-select@14`, `@ngrx/store@19`, and `ngx-infinite-scroll@19`.
 
+For more information, see the official [Angular Update Guide](https://angular.dev/update-guide?v=18.0-19.0&l=3) for updating from version 18 to version 19.
 
-## Run Spartacus update
+## Updating Spartacus to 2211.36
 
-After successfully updating the application to Angular 19, execute this command to initiate the Spartacus update process.
+The update to Spartacus 2211.36 is mostly focused on updating the framework to Angular 19. With framework updates, there is always the chance that breaking changes could be introduced for your application. In this case, additional work on your side may be required to fix issues that result from updating from 2211.32.1 to 2211.36.
 
-```bash
-ng update @spartacus/schematics@2211.35
-```
+**Note:** You must start with a version 2211.32.1 Spartacus app to be able to update to version 2211.36.
 
-### If using Server Side Rendering (SSR) and `application` builder
+1. To update to version 2211.36 of Spartacus, run the following command in the workspace of your Angular application:
 
-For applications with SSR support you need to adjust the `server.ts`:
+   ```bash
+   ng update @spartacus/schematics@2211.36
+   ```
 
-```diff
-/* ... */
-- const indexHtml = join(browserDistFolder, 'index.html');
-+ const indexHtml = join(serverDistFolder, 'index.server.html');
-```
+1. Consult [Technical Changes in Spartacus 2211.36](./typescript-manual.doc.md) for information about additional changes that have been introduced in Spartacus 2211.36.
+1. If your app uses server-side rendering (SSR) and the `application` builder, you also need to adjust the `server.ts` file by removing the following line:
 
-## Adjust the Bootstrap usage
+   ```ts
+   const indexHtml = join(browserDistFolder, 'index.html');
+   ```
 
-Spartacus internalized the Bootstrap 4 styles, so you don't need the Bootstrap being installed in your project anymore.
+1. Continuing for apps that use SSR and the `application` builder, finish adjusting the `server.ts` file by adding the following line:
 
-Please follow the guide: [Spartacus migration - Bootstrap](./bootstrap.md)
+   ```ts
+   const indexHtml = join(serverDistFolder, 'index.server.html');
+   ```
```

Comment:
AFAIK this migration step is already covered by our migration schematics automatically.
So I can see 2 options:
a) don't mention this manual step in the docs
b) tell, that it should have been done by the automated migrations. and this is just a fallback manual step notice, in case the automated migration failed due to any reason

So to avoid confusion, I think we should mention that this step should be already done in their server.ts.

Reply:
For the record, we decided to keep the overview sections for Bootstrap and silencing Sass deprecation warnings, and everything else was removed.